5-VM-Ubuntu-1-NodeTypes-Secure/azuredeploy.json

```diff
@@ -83,155 +96,212 @@
       }
     },
-    "clusterProtectionLevel": {
+    "storageAccountType": {
       "type": "string",
       "allowedValues": [
-        "None",
-        "Sign",
-        "EncryptAndSign"
+        "Standard_LRS",
+        "Standard_GRS"
       ],
-      "defaultValue": "EncryptAndSign",
+      "defaultValue": "Standard_LRS",
       "metadata": {
-        "description": "Protection level.Three values are allowed - EncryptAndSign, Sign, None. It is best to keep the default of EncryptAndSign, unless you have a need not to"
+        "description": "Replication option for the VM image storage account"
       }
     },
-    "nt0InstanceCount": {
-      "type": "int",
-      "defaultValue": 5,
+    "supportLogStorageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS"
+      ],
+      "defaultValue": "Standard_LRS",
       "metadata": {
-        "description": "Instance count for node type"
+        "description": "Replication option for the support log storage account"
       }
     },
-    "nodeDataDrive": {
+    "applicationDiagnosticsStorageAccountType": {
       "type": "string",
-      "defaultValue": "Temp",
       "allowedValues": [
-        "OS", "Temp"
+        "Standard_LRS",
+        "Standard_GRS"
       ],
+      "defaultValue": "Standard_LRS",
       "metadata": {
-        "description": "The drive to use to store data on a cluster node."
+        "description": "Replication option for the application diagnostics storage account"
       }
     },
-    "nodeTypeSize": {
+    "nt0InstanceCount": {
+      "type": "int",
+      "defaultValue": 5,
+      "metadata": {
+        "description": "Instance count for node type"
+      }
+    },
+    "vmNodeType0Size": {
       "type": "string",
       "defaultValue": "Standard_D2_v2",
       "metadata": {
-        "description": "The VM size to use for cluster nodes."
+        "description": "VM Type"
       }
     }
   },
   "variables": {
-    "vmNodeType0Name": "nodetype0-vm",
-    "vmNodeType0Size": "[parameters('nodeTypeSize')]",
+    "computeLocation": "[parameters('clusterLocation')]",
+    "dnsName": "[parameters('clusterName')]",
+    "vmStorageAccountName": "[toLower(concat( 'sflogs', uniqueString(resourceGroup().id), '1' ))]",
+    "vmName": "vm",
+    "publicIPAddressName": "PublicIP-VM",
+    "publicIPAddressType": "Dynamic",
+    "vmStorageAccountContainerName": "vhds",
     "virtualNetworkName": "VNet",
+    "addressPrefix": "10.0.0.0/16",
     "nicName": "NIC",
-    "lbIp0Name": "PublicIP-LB-FE-0",
+    "lbName": "LoadBalancer",
+    "lbIPName": "PublicIP-LB-FE",
+    "availSetName": "AvailabilitySet",
+    "maxPercentUpgradeDomainDeltaUnhealthyNodes": "100",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "overProvision": "false",
+    "vmssApiVersion": "2017-03-30",
```

Comment:
remove these and use literal values for apiVersions on the resources - this prevents schema validation from being done